dev/io.openliberty.jakarta.cdi.4.0/bnd.bnd

```diff
@@ -11,55 +11,66 @@
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion=1.0
 
+javac.source: 11
+javac.target: 11
 
-Bundle-SymbolicName: io.openliberty.jakarta.cdi.4.0; singleton:=true
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
 
-Export-Package: jakarta.decorator; version=3.0, \
- jakarta.enterprise.context; version=3.0, \
- jakarta.enterprise.context.control; version=3.0, \
- jakarta.enterprise.context.spi; version=3.0, \
- jakarta.enterprise.event; version=3.0, \
- jakarta.enterprise.inject; version=3.0, \
- jakarta.enterprise.inject.literal; version=3.0, \
- jakarta.enterprise.inject.se; version=3.0, \
- jakarta.enterprise.inject.spi; version=3.0, \
- jakarta.enterprise.inject.spi.configurator; version=3.0, \
- jakarta.enterprise.util; version=3.0, \
- jakarta.inject; version=2.0.1
+Bundle-SymbolicName: io.openliberty.jakarta.cdi.4.0; singleton:=true
 
+#these exports are copied from the manifest files of the original bundles
+Export-Package: \
+ jakarta.decorator;version="3.0";uses:="jakarta.enterprise.inject",\
+ jakarta.enterprise.context;version="3.0";uses:="jakarta.enterprise.util,jakarta.inject",\
+ jakarta.enterprise.context.control;version="3.0";uses:="jakarta.enterprise.context,jakarta.interceptor",\
+ jakarta.enterprise.context.spi;version="3.0",\
+ jakarta.enterprise.event;version="3.0";uses:="jakarta.enterprise.util",\
+ jakarta.enterprise.inject;version="3.0";uses:="jakarta.enterprise.context,jakarta.enterprise.inject.spi,jakarta.enterprise.util,jakarta.inject",\
+ jakarta.enterprise.inject.build.compatible.spi;version="3.0";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.lang.model,jakarta.enterprise.lang.model.declarations,jakarta.enterprise.lang.model.types",\
+ jakarta.enterprise.inject.literal;version="3.0";uses:="jakarta.enterprise.util,jakarta.inject",\
+ jakarta.enterprise.inject.se;version="3.0";uses:="jakarta.enterprise.inject,jakarta.enterprise.inject.spi",\
+ jakarta.enterprise.inject.spi;version="3.0";uses:="jakarta.el,jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi.configurator,jakarta.interceptor",\
+ jakarta.enterprise.inject.spi.configurator;version="3.0";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.util",\
+ jakarta.enterprise.util;version="3.0",\
+ jakarta.inject;version="2.0.1",\
+ jakarta.enterprise.lang.model;version="4.0";uses:="jakarta.enterprise.lang.model.declarations,jakarta.enterprise.lang.model.types",\
+ jakarta.enterprise.lang.model.declarations;version="4.0";uses:="jakarta.enterprise.lang.model,jakarta.enterprise.lang.model.types",\
+ jakarta.enterprise.lang.model.types;version="4.0";uses:="jakarta.enterprise.lang.model,jakarta.enterprise.lang.model.declarations"
```

Comment:
version should be 3.0

Reply:
I opened the jar and found all of the version are 3.0. I thought they would be 4.0, same as the bundle version, because they did not use semantic versioning policy.

Reply:
I will follow this up with Jakarta CDI spec.